internal/gcs/container.go

```diff
@@ -0,0 +1,171 @@
+package gcs
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/Microsoft/hcsshim/internal/cow"
+	"github.com/Microsoft/hcsshim/internal/logfields"
+	"github.com/Microsoft/hcsshim/internal/schema1"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	hrComputeSystemDoesNotExist = 0xc037010e
```

Comment:
Can we use the errors package? Do we need to redefine these?

Reply:
We could, but I think it's clearer to treat this as a magic number that is part of the protocol and not as an error. E.g. if we used this code on Linux, we would not want to rely on values of type syscall.Errno.

Reply:
It's just two typedefs. I don't really care.